File: ejercicios/python/ejercicio055.py
def potencia_iterativa(base, exponente):
   potencia = 1
   for i in range(0,exponente):
      potencia = potencia * base
   return potencia

if __name__ == "__main__":
   base = int(input("ingrese la base : "))
   exponente = int(input("ingrese el exponente : "))
   if (exponente >= 0):
      print (str(base) + " elevado a " + str(exponente) + " es : " + str(potencia_iterativa(base, exponente)))
   else:
      print("Error en exponente " + str(exponente) + " debe ser positivo.")
      

File: ejercicios/c++/ejercicio055.cpp
#include <iostream>
using namespace std;

long potencia_iterativa(int base, int exponente){	long potencia = 1;	for(int i = 0; i < exponente; i++){		potencia = potencia * base;	}	return potencia;
}

int main() {
	int base, exponente;
	cout << "ingrese la base : ";
	cin >> base;
	cout << "ingrese el exponente : ";
	cin >> exponente;
	if (exponente >= 0)
		cout << base << " elevado a " << exponente << " es : " << potencia_iterativa(base, exponente) << endl;
	else
		cout << "Error en exponente " << exponente << " debe ser positivo." << endl;
	return 0;
}



